app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,26 @@
+.banner {
+  background-color: $orange;
+  display: block;
+  height: 30px;
+  width: auto;
+}
+
+.main-section {
+  align-items: center;
+  display: flex;
+  flex-flow: row wrap;
+  justify-content: center;
+  margin: 40px;
```

Comment:
use padding instead of margin here, padding should be used to separate an element (in this case the main section) from its children, while margin should be used to separate siblings